config/default.yml

```diff
@@ -98,6 +98,14 @@ Rails/ApplicationRecord:
   VersionAdded: '0.49'
   VersionChanged: '2.5'
 
+Rails/ArelStar:
+  Description: 'Enforces `Arel.star` instead of `"*"` for expanded columns.'
+  Enabled: true
+  Safe: false
```

Comment:
Can you write a reason to description of the cop why it is unsafe?

Reply:
Sure. I will add this description in here:

> This cop is unsafe because it checks for `["*"]`.
```
hsh = { "*" => true }
hsh["*"] # This would get corrected to `hsh[Arel.star]`
```

Reply:
I get it, thank you for the address!

Reply:
@koic I added the description to the main cop, is that ok?

Reply:
Based on @eugeneius's review here https://github.com/rubocop-hq/rubocop-rails/pull/344#pullrequestreview-501581547 I removed this description. I think it is still a good idea to call this an unsafe cop, but I'm open to feedback here.

Reply:
Thank you for the update. If my understanding is correct, there seems to be concern about incompatible behavior rather than false positives. If so, `SafeAutoCorrect: false` would be more eligible than `Safe: false`.

```diff
-Safe: false
+SafeAutoCorrect: false
```

https://docs.rubocop.org/rubocop/0.92/usage/auto_correct.html#safe-auto-correct

Reply:
@koic sounds good to me, and thanks for the clarification. I've pushed this up!